app/javascript/components/header/Header.jsx

```diff
@@ -1,13 +1,19 @@
-import React from 'react';
+import React, {useState} from 'react';
 import HamburgerLogo from '../../assets/hamburger-icon.svg';
 import LogoHeader  from '../../assets/sumus-logo-final.png';
+import LogoHeaderClose from '../../assets/logo-header-close.svg';
 import LogoShop from '../../assets/shop-icon.png';
 import header from './header.scss';
 
-const Header = () => {
+const Header = ({actionOpen, isOpen}) => {
+
   return (
     <header className='header'>
-      <a className='header__hamburger_icon'><img src={HamburgerLogo} alt="Menu" /></a>
+      <a className={isOpen ? 'header__hamburger-icon-close' :'header__hamburger_icon'}
+        onClick={actionOpen}>
+        <img onClick = {actionOpen}
+          src={isOpen ? LogoHeaderClose : HamburgerLogo} alt="Menu" />
+      </a>
```

Comment:
change this tag, if you don't need an `href` you don't have to use the `a` tag

Reply:
i will now using span for this